diff --git a/config/config.go b/config/config.go
index 4e69030..f4940ab 100644
--- a/config/config.go
+++ b/config/config.go
@@ -410,6 +410,12 @@ func initConsensusProtocols() {
 	// ConsensusV17 points to 'final' spec commit
 	v17 := v16
 	v17.ApprovedUpgrades = map[protocol.ConsensusVersion]bool{}
+
+	v17.UpgradeVoteRounds = 2
+	v17.UpgradeThreshold = 1
+	v17.UpgradeWaitRounds = 2
+	v17.ApprovedUpgrades[protocol.ConsensusFuture] = true
+
 	Consensus[protocol.ConsensusV17] = v17
 
 	// v16 can be upgraded to v17.
@@ -427,7 +433,7 @@ func initConsensusProtocols() {
 
 	// ConsensusFuture is used to test features that are implemented
 	// but not yet released in a production protocol version.
-	vFuture := v18
+	vFuture := v17
 	vFuture.TxnCounter = true
 	vFuture.Asset = true
 	vFuture.LogicSigVersion = 1
